lib/linalg/BasisGenerator.h

```diff
@@ -142,20 +143,39 @@ class BasisGenerator
         }
     }
 
+    /**
+     * @brief Load previously saved sample (basis or state)
+     *        within a column range.
+     *
+     * @param[in] base_file_name The base part of the name of the files
+     *                           holding the basis / snapshot vectors.
+     * @param[in] kind A string equal to "basis" or "snapshot", representing
+     *                 which kind of data to load.
+     * @param[in] col_min The first basis or snapshot to read.
```

Comment:
Comment that the column indices are 1-based? Or should we change to 0-based? I suppose this is a rare case where 1-based is more natural, so we can keep it as 1-based.